llama_stack/apis/files/files.py

```diff
@@ -77,6 +77,38 @@ class ListFileResponse(BaseModel):
     data: List[FileResponse]
 
 
+@json_schema_type
+class BucketRegistrationResponse(BaseModel):
+    """
+    Response after registering a storage location.
+
+    :param bucket: The registered storage location URI (e.g., "s3://my-bucket" or "file:///data")
+    :param created_at: Timestamp of registration
+    :param status: Current status of the storage location
+    """
+
+    bucket: str
+    created_at: int
```

Comment:
should this be `datetime`?

Reply:
mouarf got misled by the current `FileResponse` :/